src/Components/FormCreation/CardComponents/FileQuestion.tsx

```diff
@@ -38,16 +39,51 @@ function FileQuestion({
   const numFileOptions = [1, 2, 3, 5, 10];
 
   async function handleFile(event: any) {
-    let file = event.target.files[0]; //access the file
-    //todo: check if the file extension is valid (on submission)
+    const file = event.target.files[0]; //access the file
+    //todo: check if the file extension is valid
     const formData = new FormData();
     formData.append("file", file); // appending file
-    //result will be a link to the file, which we should save to our DB
-    await FILE.fileUpload("arbitrium", file.name, formData);
+
+    //returns the url to the file in aws
+    await fileUpload("arbitrium", file.name, formData);
+    //TODO Save to DB
+  }
+
+  //download a file from AWS
```

Comment:
These functions are not in use yet - I will complete the testing in the submissions pr. Wanted to avoid merge conflicts